src/client/id_detection.go

```diff
@@ -0,0 +1,55 @@
+package client
+
+import (
+	"strings"
+
+	"github.com/newrelic/nri-kafka/src/args"
+	"github.com/newrelic/nri-kafka/src/connection"
+)
+
+const (
+	consumerDetectionPattern = "kafka.consumer:type=consumer-fetch-manager-metrics,client-id=*"
+	producerDetectionPattern = "kafka.producer:type=producer-metrics,client-id=*"
```

Comment:
The _detection patterns_ currently used were already used to fetch metric. We could use `app-info` instead. Ex: `kafka.consumer:type=app-info,id=*`.  WDYT?

Reply:
What would we win with that?

Reply:
Would it be more generic?

Reply:
As @paologallinaharbur suggested, I will check both, first I will look for `app-info` and then I will look for the current one a s fallback. Thanks!

Reply:
Great idea.

Reply:
I updated it here [a4f33bf](https://github.com/newrelic/nri-kafka/pull/178/commits/a4f33bf144032481298c9d7d607d6156b824af17), could you review it again please?